### ISSUE TITLE
Add MPFD

### ISSUE THREAD

**Issue by strogonoff** [riboseinc/metanorma.com#31]

**Comment by ronaldtse** [comment]
@strogonoff I see the addition, but could you help separate this out as an "Enterprise" adoption just like the Ribose style, rather than a "Standards body" adoption? Thanks!

**Comment by strogonoff** [comment]
Thanks for reminding about that, it would be the correct way.

Things I am considering are potentially reusing software listing rather than having the same info duplicated in a hard-coded table with supported processors, keeping the complexity of the getting-started section in check and splitting some of the info into separate sections of metanorma.com, also rethinking navigation (possibly just updating OP theme ToC navigation so that it can work across other project site sections and not just docs).

**Comment by ronaldtse** [comment]
Regarding duplication of content, the key point is a person that writes a CalConnect document will not want to know how to write an UNECE document. It is necessary to have tailored content for each organization because the sample content will be different and how they achieve things are also different.

**Comment by strogonoff** [comment]
I think I understand that. We still will have a list and the question is how the list is put together, whether it is duplicated in different places which go out of sync w/each other or is in one place, and how easy it is to find. It’s going to get better soon I think:)

**Comment by opoudjis** [comment]
@strogonoff: The duplication of content is currently an area of aggravation for me, because I am the one writing the documentation, and I am refusing to write the same documentation 10 times for different audiences. (The 10 times is not an exaggeration, that's how many standards we are maintaining.) I have tried to genericise the existing documentation so that it can be repurposed for other standards. Someone may well need to managing remixed documentation through includes. I would rather that someone not be me.

**Comment by strogonoff** [comment]
@opoudjis This particular moment was about the list of Metanorma processors (classes?), which can be entirely automatically generated given we have a data structure containing the requisite info.

As to the problem you mentioned, we’re approaching the solution. I encourage to keep things as is (document just the Metanorma-ISO gem and keep that documentation up-to-date instead of duplicating it for other gems) until those docs get refactored (likely out of gem and into part of metanorma.com itself, to which gem docs will link for common docs).

**Comment by ronaldtse** [comment]
@strogonoff is MPFD now shown or hidden on the metanorma.com site?

As mentioned:

1. Let's have a separate "Enterprise" page that lists out all the enterprise flavors.
2. For standard bodies, let's have them on the front page to allow the audience to click on them ASAP on landing.

**Comment by strogonoff** [comment]
@ronaldtse 

1. You mean https://www.metanorma.com/software/user:enterprises/?
2. Sure, as discussed on the call!

**Comment by ronaldtse** [comment]
@strogonoff yes, but could we have a page that has a list of "document classes" rather than a list of implementations?

Such as:

* MPFA
* Ribose
* ...
* Acme

Maybe at a URL like www.metanorma.com/enterprises

**Comment by strogonoff** [comment]
Sure, we can do something like that
